app/src/main/res/layout/topic_practice_header_view.xml

```diff
@@ -21,18 +23,20 @@
       android:text="@string/topic_practice_master_these_skills"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="20sp"
+      android:layout_marginEnd="88dp"
```

Comment:
You can give margin end as 32 dp for consistency.

Reply:
Done